src/neo/Plugins/Plugin.cs

```diff
@@ -85,16 +85,20 @@ private static Assembly CurrentDomain_AssemblyResolve(object sender, ResolveEven
             if (args.Name.Contains(".resources"))
                 return null;
 
+            AssemblyName an = new AssemblyName(args.Name);
+
             Assembly assembly = AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(a => a.FullName == args.Name);
-            if (assembly != null)
-                return assembly;
+            if (assembly is null)
+                assembly = AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(a => a.GetName().Name == an.Name);
```

Comment:
Can we join this two linq queries in only one?

Reply:
No. They are two different queries.